modules/etc/commands/tx.go

```diff
@@ -0,0 +1,62 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/tendermint/basecoin/client/commands"
+	"github.com/tendermint/basecoin/client/commands/txs"
+	"github.com/tendermint/basecoin/modules/etc"
+)
+
+// SetTxCmd is CLI command to set data
+var SetTxCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Sets a key value pair",
+	RunE:  commands.RequireInit(setTxCmd),
+}
+
+// RemoveTxCmd is CLI command to remove data
+var RemoveTxCmd = &cobra.Command{
+	Use:   "Remove",
+	Short: "Removes a key value pair",
+	RunE:  commands.RequireInit(removeTxCmd),
+}
+
+//nolint
```

Comment:
cuz i was too lazy to comment both exported const.
otherwise the automatic linter complains :(